after/main.py

```diff
@@ -1,56 +1,53 @@
-import hydra
 import torch
-from hydra.core.config_store import ConfigStore
-from omegaconf import OmegaConf
```

Comment:
jsonargparse could be configured to use OmegaConf as the yaml parser to have its variable interpolation features, see https://jsonargparse.readthedocs.io/en/stable/#variable-interpolation.